docs/ContribOperators.md

```diff
@@ -2243,30 +2243,31 @@ This version of the operator has been available since version 1 of the 'com.micr
 #### Version
 
 This version of the operator has been available since version 1 of the 'com.microsoft' operator set.
+This function computes the inverse of the one-dimensional n-point RFFT computed in 'com.microsoft.rfft'.
 
 #### Attributes
 
 <dl>
 <dt><tt>normalized</tt> : int</dt>
-<dd></dd>
+<dd>must be 0, normalization currently not supported</dd>
```

Comment:
Is this op generated? Do we need an update to the op definition?

Reply:
https://github.com/microsoft/onnxruntime/blob/eeef15788804b0217d46ad6d1c2744585ad98d0c/onnxruntime/core/graph/contrib_ops/contrib_defs.cc#L1253-L1262 this seems to be the place

Reply:
Thanks for the catch @justinchuby  . I missed this. Updated the contrib_defs and reverted the markdown file.

The build still passes on my system.